x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
Is any of this hooks logic worth splitting into smaller functions and unit testing?